application/views/admin/push_notifications.php

```diff
@@ -20,7 +20,7 @@
                                     <div class="form-group">
                                             <label for="visibility">Visibility :</label>
                                             <select class="form-control" id="visibility" name="visibility">
-                                                <option value="null">Whole Site</option>
+                                                <option value="0">Whole Site</option>
                                                 <?php
                                                 foreach ($sessions as $session)
                                                 { ?>
```

Comment:
Why no sessions selector? This has to be sessions specific, means should be able to send push notifications only to a particular session eg; 25